### DISCUSSION TITLE
About UNes

### DISCUSSION THREAD

**Discussion by PetarGlomuz** [ls9512/UNES#3] [Q&A]
I downloaded your project and check with my nes files.
It is working well, but only the problem is Audio.
Could you please share if you will make APU feature also?

**Comment by ls9512** [comment]
This is a transplant project, the original project did not implement the APU part, and the sound playback depends on the platform and hardware, which is different from the display rendering that can use Unity's cross-platform capabilities.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ls9512/UNES"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月8日(星期一) 中午11:11
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[ls9512/UNES] About UNes (Discussion #3)





 
I downloaded your project and check with my nes files.
 It is working well, but only the problem is Audio.
 Could you please share if you will make APU feature also?
